doc/sedcli.8

```diff
@@ -94,6 +94,18 @@ MBR shadow should be enabled for the PBA to take effect.
 .IP "\fB\-B -d <device> -r {1|0}\fR or \fB\-\-block_sid --device <device> --hwreset {1|0}\fR"
 Issue BlockSID authentication command with Clear Event flag via --hwreset option.
 
+.IP "\fB\-U -d <device> -u <userN> -t <accesstype> [-l <locking-range>]\fR or \fB\-\-add-user-to-lr --device <device> --user <userN> --accesstype {RO|RW|LK} [--locking-range <locking-range>]\fR"
+.IP
+Add users to the Locking Ranges with the following access modes:
```

Comment:
I don't think there is such thing as each (user) authority has its own access mode. I need to confirm that though.

Reply:
It is possible to set/clear: ReadLockEnabled, WriteLockEnabled, ReadLocked, WritedLocked and many other things int Locking Table. Given locking object (row in a Locking Table) can associated with user object. More details are in section 5.7.2.2. or TCG_storage_architecture_core spec...